Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/contactdiary/ui/ContactDiaryCommentInfoTestFragment.kt

```diff
@@ -0,0 +1,31 @@
+package de.rki.coronawarnapp.test.contactdiary.ui
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.ContactDiaryCommentInfoFragmentBinding
+import de.rki.coronawarnapp.test.menu.ui.TestMenuItem
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+
+class ContactDiaryCommentInfoTestFragment : Fragment(R.layout.contact_diary_comment_info_fragment) {
```

Comment:
very minor: Maybe we should start naming it properly --> ContactJournal instead of Diary?